pkgs/by-name/co/code-nautilus/package.nix

```
@@ -0,0 +1,39 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nautilus-python,
+  python3,
+}:
+stdenv.mkDerivation rec {
+
+  pname = "code-nautilus";
+  version = "8ea0ce78f3f1f7a6af5f9e9cf93fc3e70015f61e";
```

```suggestion
  version = "0-unstable-YYYY-MM-DD";
```
with the commit date